adapters/huaweiads/huaweiadstest/exemplary/banner5_user_geo.json

```diff
@@ -0,0 +1,272 @@
+{
+  "mockBidRequest": {
```

Comment:
You already have a test (`banner1.json`) that adds coverage for `user.geo.country` so you can safely remove this test

Reply:
https://github.com/prebid/prebid-server/pull/2345/files#r964777620

Because of this comment, we added the new test file. If it is already covered, we will delete the file

Reply:
Also this file covers the second if check of the getCountryCode function, by only having user.geo.country and no devide.geo.country and device.mccmnc

Reply:
@mansinahar 

banner1.json covers `device.geo.country`
banner5_user_geo.json covers `user.geo.country`

This is added due to this comment https://github.com/prebid/prebid-server/pull/2345/files#r964777620